pkg/config/state_root.go

```diff
@@ -1,6 +1,11 @@
 package config
 
+import "time"
+
 // StateRoot contains state root service configuration.
 type StateRoot struct {
-	Enabled bool `yaml:"Enabled"`
+	Enabled         bool          `yaml:"Enabled"`
+	ResponseTimeout time.Duration `yaml:"ResponseTimeout"`
+	Nodes           []string      `yaml:"Nodes"`
```

Comment:
We now have some "Vote" messages for that, see neo-project/neo-modules#512.

Reply:
`Vote` is a replacement for RPC signature sending, not related to nodes.

Reply:
Well, it's related because now `Nodes` are unused.